app/src/main/kotlin/hr/asee/android/template/compose/ui/prelogin/register/model/RegisterMessages.kt

```diff
@@ -0,0 +1,29 @@
+package hr.asee.android.template.compose.ui.prelogin.register.model
+
+import hr.asee.android.template.compose.R
+import hr.asee.android.template.compose.ui.common.model.Message
+
+enum class RegisterMessages : Message {
+    MISSING_PASSWORD_ERROR,
+    STORE_ACCESS_TOKEN_ERROR,
+    USER_NOT_FOUND_ERROR,
+    MISSING_EMAIL_OR_USERNAME_ERROR;
```

Comment:
znam da si samo kopirao pa ti je vjerojatno promaklo, ali ova greške treba prilagoditi registraciji, to možeš popraviti zajedno s implementacijom backend poziva kad vidiš koje se greške mogu javljati pa sukladno tome ih ovdje popisati